.github/workflows/cmake.yml

```diff
@@ -117,4 +117,4 @@ jobs:
     - name: Unit Test
       working-directory: ${{github.workspace}}/build
       shell: bash
-      run: ctest -j8 --output-on-failure -C $BUILD_TYPE
+      run: sudo ctest -j8 --output-on-failure -C $BUILD_TYPE
```

Comment:
I have a bit concern of sudo permission to run this test.

Reply:
Done. `sudo` has been removed, and the way for creating tars has been slightly adjusted.